tests/common/helpers/ptf_tests_helper.py

```diff
@@ -0,0 +1,171 @@
+"""
+A helper module for PTF tests.
+"""
+
+import pytest
+import random
+
+from ipaddress import ip_address, IPv4Address
+from tests.common.config_reload import config_reload
+
+
+@pytest.fixture(scope="module")
+def downstream_links(duthost, tbinfo):
+    """
+    Returns a dictionary of all the links that are downstream from the DUT.
+
+    Args:
+        duthost: DUT fixture
+        tbinfo: testbed information fixture
+    Returns:
+        links: Dictionary of links downstream from the DUT
+    """
+    links = dict()
+
+    def filter(interface, neighbor, mg_facts, tbinfo):
+        if ((tbinfo["topo"]["type"] == "t0" and "Server" in neighbor["name"])
+                or (tbinfo["topo"]["type"] == "t2" and "T1" in neighbor["name"])):
```

Comment:
What about if topo is T1? 
The test case should be run on T0 and T1. We can skip T2 for now.